dotcom-rendering/index.d.ts

```diff
@@ -513,6 +513,123 @@ interface CAPIArticleType {
 	mostRecentBlockId?: string;
 }
 
+interface FrontType {
+	pressedPage: PressedPageType;
+	nav: CAPINavType;
+	editionId: string;
+	editionLongForm: string;
+	guardianBaseURL: string;
+	pageId: string;
+	webTitle: string;
+	webURL: string;
+	config: FrontConfigType;
+	commercialProperties: Record<string, unknown>;
+}
+
+type PressedPageType = {
+	id: string;
+	seoData: SeoDataType;
+	frontProperties: FrontPropertiesType;
+	collections: PressedCollectionType[];
+};
+
+type SeoDataType = {
+	id: string;
+	navSection: string;
+	webTitle: string;
+	description: string;
+};
+
+type FrontPropertiesType = {
+	isImageDisplayed: boolean;
+	commercial: Record<string, unknown>;
+};
+
+type PressedCollectionType = Record<string, unknown>;
+
+type FrontConfigType = {
+	avatarApiUrl: string;
+	externalEmbedHost: string;
+	ajaxUrl: string;
+	keywords: string;
+	revisionNumber: string;
+	isProd: boolean;
+	switches: Switches;
+	section: string;
+	keywordIds: string;
+	locationapiurl: string;
+	sharedAdTargeting: { [key: string]: any };
```

Comment:
I see we’ve used `Record<string, unknown>` above. We might want to keep these consistent throughout?